atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraRequestExceptionHandlerTest.java

```diff
@@ -44,12 +46,23 @@
     private static final int MAX_RETRIES_PER_HOST = 3;
     private static final int MAX_RETRIES_TOTAL = 6;
 
-    private static Set<Exception> connectionExceptions = Sets.newHashSet(new SocketTimeoutException(MESSAGE),
+    private static final Set<Exception> CONNECTION_EXCEPTIONS = ImmutableSet.of(new SocketTimeoutException(MESSAGE),
             new CassandraClientFactory.ClientCreationFailedException(MESSAGE, CAUSE));
-    private static Set<Exception> transientExceptions = Sets.newHashSet(new TTransportException());
-    private static Set<Exception> indicativeOfCassandraLoadException = Sets.newHashSet(new NoSuchElementException(),
-            new TimedOutException(), new UnavailableException(), new InsufficientConsistencyException(MESSAGE));
-    private static Set<Exception> fastFailoverExceptions = Sets.newHashSet(new InvalidRequestException());
+    private static final Set<Exception> TRANSIENT_EXCEPTIONS = ImmutableSet.of(new TTransportException());
+    private static final Set<Exception> INDICATIVE_OF_CASSANDRA_LOAD_EXCEPTIONS =
+            ImmutableSet.of(
+                    new NoSuchElementException(),
```

Comment:
This is thrown by the client pool, not cassandra, right? Could also be indicative of too high query volume or over parallelization, rather than c* load.